### YamlMime:JavaMember
uid: "org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap.setExceptionCallback*"
fullName: "org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap.setExceptionCallback"
name: "setExceptionCallback"
nameWithType: "H2MultiplexingRequesterBootstrap.setExceptionCallback"
members:
- uid: "org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap.setExceptionCallback(org.apache.hc.core5.function.Callback<java.lang.Exception>)"
  fullName: "org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap.setExceptionCallback(Callback<Exception> exceptionCallback)"
  name: "setExceptionCallback(Callback<Exception> exceptionCallback)"
  nameWithType: "H2MultiplexingRequesterBootstrap.setExceptionCallback(Callback<Exception> exceptionCallback)"
  summary: "Assigns <xref uid=\"java.lang.Exception\" data-throw-if-not-resolved=\"false\" data-raw-source=\"Exception\"></xref> <xref uid=\"org.apache.hc.core5.function.Callback\" data-throw-if-not-resolved=\"false\" data-raw-source=\"Callback\"></xref> instance."
  parameters:
  - name: "exceptionCallback"
    type: "<xref href=\"org.apache.hc.core5.function.Callback?alt=org.apache.hc.core5.function.Callback&text=Callback\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"java.lang.Exception?alt=java.lang.Exception&text=Exception\" data-throw-if-not-resolved=\"False\" />&gt;"
  syntax: "public final H2MultiplexingRequesterBootstrap setExceptionCallback(Callback<Exception> exceptionCallback)"
  returns:
    type: "<xref href=\"org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap?alt=org.apache.hc.core5.http2.impl.nio.bootstrap.H2MultiplexingRequesterBootstrap&text=H2MultiplexingRequesterBootstrap\" data-throw-if-not-resolved=\"False\" />"
type: "method"
metadata: {}
package: "org.apache.hc.core5.http2.impl.nio.bootstrap"
artifact: com.azure:azure-security-keyvault-jca:1.0.0-beta.2
